### PR TITLE
Aria Logger Fix

### PR DESCRIPTION
**Pull request by xTEddie**
- Fix `AriaTelemetryLogger` to be initialized to pick the proper `collectorUI` on every logging event
- Update `environmentVersion` to be `prod` by default in `defaultInternalTelemetryData`

### CODE REVIEW

File: chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { IInternalTelemetryData } from "../interfaces/IInternalTelemetryData";
 
 export const defaultInternalTelemetryData: IInternalTelemetryData = {
-    environmentVersion: "test",
+    environmentVersion: "prod",
```
Comment (elopezanaya) [inline chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts:4]: can we add this change to the change.log?, to communicate the switch from test to prod as default

Comment (akhiloct1510) [inline chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts:4]: Will this start pushing all the telemetry to prod instances irrespective of environment, like ocdailym telemetry flowing to prod

Comment (xTEddie) [reply chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts:4]: Telemetry is already pushed to prod by default. `environmentVersion` is a hardcoded value. The logger has a logic to ignore `collectorURI` if `environmentVersion` is not set to `prod`

File: chat-widget/src/common/telemetry/loggers/ariaTelemetryLogger.ts [hunk 2]
```diff
@@ -8,14 +8,30 @@ import { Constants, AriaTelemetryConstants, EnvironmentVersion } from "../../Con
 import { IChatSDKLogger } from "../interfaces/IChatSDKLogger";
 import { TelemetryManager } from "../TelemetryManager";
 
+const AWTDefaultConfiguration = {
+    collectorUri: "https://browser.pipe.aria.microsoft.com/Collector/3.0/",
+    cacheMemorySizeLimitInNumberOfEvents: 10000,
+    disableCookiesUsage: false,
+    canSendStatEvent: (eventName: string) => { return true; }, // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (kevinlin98z) [inline chat-widget/src/common/telemetry/loggers/ariaTelemetryLogger.ts:15]: What is this being used for?

Comment (xTEddie) [reply chat-widget/src/common/telemetry/loggers/ariaTelemetryLogger.ts:15]: I believe it's the callback to whether to send telemetry. It's the default configuration from Aria that was reused.

**Pull request merged**